### DISCUSSION TITLE
Is it a good practice to use suffix along with prefix?

### DISCUSSION THREAD

**Discussion by therealharpaljadeja** [celo-org/social-connect#248] [Help (Q&A)] [answered]
Should I use suffixes along with prefixes with identifiers, is it good or there could be some problems?

**Comment by eelanagaraj** [comment, marked as answer]
Generally avoid using suffixes since this is not part of the identifier format and would lead to identifiers not being interoperable with others. Using the identity SDK (`@celo/identity`) will help ensure that identifiers are formatted in the proper way to enable discovery by anyone else also adhering to the standards 😄
